features/languages.feature

```diff
@@ -0,0 +1,54 @@
+Feature: Languages
+@QA @Languages
+Scenario: Ensure you can change languages in CommCare
+	Then I install the ccz app at "languages.ccz"
+	Then I login with username "clark" and password "123"
+
+	#Check that default language deployed
+	Then I press start
+	Then I select module "Basic Form Test"
+	And I select form "Languages"
+
+	Then I see the text "Enter a name:"
+	Then I enter text "English"

```

Comment:
Text entry isn't needed, I would remove this line (and all the other text entry ones below)